examples/org-viewer-react/src/OrgApps.tsx

```diff
@@ -0,0 +1,31 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { useApps } from '@aragon/connect-react'
+import Group from './Group'
+import Table from './Table'
+import TextButton from './TextButton'
+
+type Props = {
+  onOpenApp: (address: string) => void
+}
+
+export default function OrgApps({ onOpenApp }: Props) {
+  const [apps, { loading }] = useApps()
+
+  return (
+    <Group name="Apps" loading={loading}>
+      <Table
+        headers={['Name', 'Version', 'Address']}
+        rows={[...apps]
+          .sort((a, b) => (a.name || '').localeCompare(b.name || ''))
+          .map(app => [
```

Comment:
Not a big deal, but I think you could make this a bit smaller by doing:

```jsx
rows={apps
  .map()
  .sort()
}
```

Reply:
Yes good idea!